src/post.ts

```diff
@@ -51,6 +59,11 @@ function coldMedicineCabinet(): void {
   }
 }
 
+function floristFriars(): void {
+  if (!have() || myLocation() !== $location`Barf Mountain` || isFull()) return;
+  [StealingMagnolia, AloeGuvnor, PitcherPlant].forEach((flower) => flower.plant());
```

Comment:
make sure to have tested this; I have never personally used this code lol

Reply:
Wasn't able to test, looks like libram florist friar code is broken